caikit_nlp/modules/token_classification/filtered_span_classification.py

```diff
@@ -137,6 +137,28 @@ def run(
                         token_classification_results.append(token_classification)
         return TokenClassificationResult(results=token_classification_results)
 
+    def run_bidi_stream(self, text_stream, threshold: Optional[float] = None):
+        """Function to provide bi-directional streaming inferencing for this module"""
+
+        # TODO: For optimization implement window based approach.
+
+        for span_output in self._stream_span_output(text_stream):
+            classification_result = self.sequence_classifier.run(span_output.text)
+            for classification in classification_result.results:
+                label = classification.label
+                # TODO: Figure out how we want to handle thresholding for streaming
+                # since with streaming, if the sentence we are processing
+                # is below threshold, do we want to skip yielding that response and try to get
+                # another one ? What if none cross / passes threshold.
+                yield TokenClassification(
+                    start=span_output.span.start,
```

Comment:
I guess CI doesn't actually run yet, but hopefully once the PR this is chained to is merged
```suggestion
                    start=span_output.start,
```